src/app/DummyPage.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+import DocumentTitle from 'react-document-title'
+import { Button } from 'carbon-components-react'
+import { logo } from 'polymath-ui'
+
+import illustration from './illustration.png'
```

Comment:
Put raster images into the 'public' dir. We don't need react to handle them.

Reply:
Fixed